app/handlers/ErrorHandler.scala

```diff
@@ -31,4 +36,24 @@ class ErrorHandler @Inject()(
 
   override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit rh: Request[_]): Html =
     view(pageTitle, heading, message)
+
+  override def onClientError(request: RequestHeader, statusCode: Int, message: String): Future[Result] = {
+
+    if (statusCode == FORBIDDEN) {
+
+      implicit val rhToRequest: Request[String] = Request(request, "")
+
+      Logger.warn(s"[ErrorHandler.onClientError] Forbidden request with message: $message")
```

Comment:
Can we turn this up to Logger.error?

```suggestion
      Logger.error(s"[ErrorHandler.onClientError] Forbidden request with message: $message")
```